### ISSUE TITLE
Possible bug with antd radio

### ISSUE THREAD

**Issue by prodevelsol** [vazco/uniforms#396]
Hello, a possible bug with antd radio buttons as they are rendered correctly but not checked when it is being clicked. I connected the form to redux and here is the code for the radio button:
```
const renderRadioField = ({ type, label, input, ...rest }) => {
  return  <RadioField
    type={type}
    name={input.name}
    label={label}
    {...input}
    {...rest}
  />;
}
```

The value is being updated in the redux store and state is being updated correctly the problem is it is not displayed as "checked";

Is it something wrong with my render function?

I also use the same code for checkbox and it works fine.

**Comment by radekmie** [comment]
Hi @prodevelsol. Which version of antd? Also, can you check if it’s working with the default one? I’m using it (you can see it also in the demo), and it’s working.

**Comment by prodevelsol** [comment]
@radekmie  antd version 3.2.1, uniforms-antd and uniforms version 1.23. Tried to render a field inside a base form like this: 

```
<RadioField name="fieldName" label="Title" allowedValues={['value1', 'value2']} />
```

Also tried same code into an autoform and it works fine very weird :) I am going to check out a little bit more. Because in my Form component that is connected to redux it doesn't work and it's the only field type that doesn't work.

Also found out that this code doesn't work:
```
<BaseForm
     schema={forgotPasswordSchema}
     onSubmit={data => this.handleSubmit(data)}
 >
     <RadioField
          name="email"
          allowedValues={['mastercard', 'dsds']} transform={value => value.toUpperCase()} 
      />
      <ErrorsField />
      <SubmitField className="login-form-button" />
</BaseForm>
```

but this one works fine:
```
<AutoForm
     schema={forgotPasswordSchema}
     onSubmit={data => this.handleSubmit(data)}
 >
     <RadioField
          name="email"
          allowedValues={['mastercard', 'dsds']} transform={value => value.toUpperCase()} 
      />
      <ErrorsField />
      <SubmitField className="login-form-button" />
</AutoForm>
```

**Comment by radekmie** [comment]
If you want to bind your form state to redux, you have to use `onChange` on your form. It doesn't work, because `BaseForm` calls `onChange` and that's all. [`AutoForm` manages state by itself](https://github.com/vazco/uniforms/blob/master/INTRODUCTION.md#forms-components).

TL;DR;
Use `<BaseForm onChange={...} value={...} ...>`.

**Comment by radekmie** [comment]
No response so far, closing.
